index.js

```diff
@@ -19,23 +19,77 @@ const createRvClient = (
     );
   }
 
-  const sha1 = async (seed) => {
-    const enc = new TextEncoder();
-    const hash = await crypto.subtle.digest('SHA-1', enc.encode(seed));
-    return Array.from(new Uint8Array(hash)).
-        map(v => v.toString(16).padStart(2, '0')).
-        join('');
-  };
+  const pemToArrayBuffer = pem => {
```

Comment:
`(pem) => `

Reply:
why? not necessary

Reply:
We can add formatting on the go for better maintainability.